portfolio/src/main/webapp/index.html

```diff
@@ -221,6 +226,31 @@ <h4 class="blue-text">Places to Travel</h4>
             </div>
           </div>
         </div>
+        <!-- Trying week3 post to server form--> 
+        <form action="/data" method="POST" id="datastore-form" name="recommendation-form-server">
+          <h5 class="blue-text">
+        	  Give me your recommendations! Music, books, movies, tv-shows, foods, I'm all ears :)
+              </h5>
+          <input id="name" type="text" name="name" placeholder="Your Name?">
+          <select id="category" placeholder="category" name="category" required>
+            <option value="" disabled selected>Select a Category</option>
+            <option value="Literature">Literature</option>
+            <option value="Music">Music</option>
+            <option value="Movie">Movie</option>
+            <option value="TV-Show">TV-Show</option>
+            <option value="Travel">Travel Destination</option>
+            <option value="Food">Food</option>
+            <option value="Activity">Activity</option>
+            <option value="Other">Other</option>
+          </select>
+          <input id="recommendation" type="text" 
+              placeholder="Your Recommendation?" name="recommendation">
+          <textarea id="comment" type="text" 
+              placeholder="Any additional comments?" name="comments"></textarea>
+          <button type="submit" id="submit-form">Send</button>
+        </form>
+
+        <!--Original Form--> 
```

Comment:
Is it showing an additional form on the page?

Reply:
Yes, I'm sorry for not- I just uploaded the link to the deployed version in PR description. 

When you hit the "Suggestion Box" button, two forms will pop up. The first one is the one I added for this week's functionality- to post form to datastore. The second one is the one I had last week that posts to google sheets. I intend on combining them into one later.

Reply:
okay, noted!